keps/sig-network/1880-multiple-service-cidrs/kep.yaml

```diff
@@ -20,18 +20,18 @@ see-also:
 replaces:
 
 # The target maturity stage in the current dev cycle for this KEP.
-stage: beta
+stage: stable
```

Comment:
I forget the details.  This was beta, but off by default.  So it didn't get a ton of usage.  Is going straight to GA safe?  Will we set the lock-to-default?

Reply:
let can avoid setting the lock to default, that will allow cluster admins to disable the feature gate entirely without needing to use a webhook or VAP https://github.com/kubernetes/enhancements/pull/4983#issuecomment-2541699445 , cc: @danwinship